tasks/main.yml

```diff
@@ -21,3 +21,4 @@
 # Configure PostgreSQL.
 - import_tasks: users.yml
 - import_tasks: databases.yml
+- import_tasks: extensions.yml
```

Comment:
perhaps I should tack in a `when:` here, skipping if the extensions array is empty...?

Reply:
I would suggest to follow same pattern as for users and databases - without `when`.